src/finders/CognitoAppClientIdFinder.js

```diff
@@ -0,0 +1,70 @@
+class CognitoAppClientIdFinder {
+
+  async find(name) {
+    // Split the name on a period. The expected syntax for `name` is `<UserPoolName>.<AppClientName>`
+    const split = name.split(".")
+    const userPoolName = split[0];
+    const appClientName = split[1];
```

Comment:
You might consider either checking first to ensure the string contains a period, or checking to make sure the `split` array length is > 1 and providing an error message to the user if not. Currently if someone uses an incorrect format the whole thing will just blow up and the Serverless output won't be very helpful to the user.